docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,147 @@
+
+#### 1.5 Build
+Spusťte build, např prostřednictvím phingového targetu 
+```
+php phing build-demo-dev
+```
+Po proběhnutí buildu je již vytvořena jednodoménová aplikace
+
+### 2. Přidání nové domény
+
+#### 2.1 Konfigurace domény
+Přidejte konfiguraci nové domény do `app/config/domains.yml`.
+V tomto souboru se nastavuje ID domény, identifikátor domény pro doménové záložky v administraci, jazyk domény.
+
+#### 2.2 Konfigurace url adresy
+Nastavte url adresu nové domény v `app/config/domains_urls.yml`.
+
+*Pozn: V případě, že přidávate novou doménu na platformě Mac, musíte tuto doménu povolit i v interfasu networku, viz [Installation Using Docker for MacOS](https://github.com/shopsys/shopsys/blob/master/docs/installation/installation-using-docker-macos.md#11-enable-second-domain-optional)
+
+#### 2.3 Nastavevení projektu jako multidoménový
+Upravte nastavení parametru `is-multidomain` v `build.xml` na hodnotu `true`.
+Prostřednictvím této změny se upraví chování některých phingových targetu - např při importu demo dat se naimportují data pro další doménu v případě, že jsou k dispozici.
+
+#### 2.4 Nastavení jazyka
+Nastavení jazyka domény proveďte podle instrukcí ze sekce [Nastavení jazyka](#3-nastaveni-jazyka)
+
+#### 2.5 Nastavení multidoménových atributu 
+Pro správne fungování nové domény je nutno nastavit hodnoty multidoménových atributu pro tuto novou doménu.
+Pro nastavení hodnot multidoménových atributu spusťte phingový target
+```
+php phing create-domains-data
+```
+Při běhu se pro novou doménu nakopírují hodnoty multidoménových atributu z 1. domény, viz `Shopsys/FrameworkBundle/Component/Domain/DomainDataCreator.php`, ve kterém je nadefinována konstanta `TEMPLATE_DOMAIN_ID`.
+V případě, že je u přidávané domény nastaven i nový jazyk, dojde při spuštění targetu i k vytvoření překladových záznamu pro tento nový jazyk - tyto záznamy ale budou prázdné.
+Při běhu je pro novou doménu vytvořena i jedna cenová skupina s názvem Default.
+Součástí targetu je i spuštění automatických přepočtu cen, dostupností, a viditelností produktu.
+
+#### 2.6 Vygenerování souboru pro dizajn nové domény
+K správnemu zobrazení nové domény je nutno vygenerovat assets
+```
+php phing grunt
+```
+
+### 3. Nastavení jazyka
+V případě jazyku cz a en jsou v Shopsys Frameworku některé části z tohoto návodu již předpřipraveny.
+
+#### 3.1 Konfigurace domény
+Nastavte jazyk u konkrétní domény v `app/config/domains.yml`.
+V tomto souboru se nastavuje ID domény, identifikátor domény pro doménové záložky v administraci, jazyk domény.
+
+#### 3.2 Routy pro nový jazyk
+Vytvořte soubor s routami v `ShopBundle/Resources/config/` podle vzoru `routing_front_xx.yml` kde `xx` je kód nového jazyka.
+Novou konfiguraci pro routy je nutno naimportovat v `app/config/packages/shopsys_shop.yml`
+
+#### 3.3 Zprávy a popisky
+Pro správně zobrazení popisku typu *Registrace*, *Košík*, vytvořte pro nový jazyk překlady zpráv v `ShopBundle/Resources/translations/`.
```

Comment:
it is not clear, what I need to do here. I would add more detailed instructions - I am not sure whether running `php phing dump-translations` is sufficient or whether you have to manually create all the `*.po` files?